tests/Microsoft.Bot.Builder.Tests/StorageBaseTests.cs

```diff
@@ -113,10 +113,10 @@ protected async Task UpdateObjectTest(IStorage storage)
             }
 
             // write with old etag should FAIL for storeitem
-            
+
             updatePocoStoreItem.Count = 123;
 
-            await Assert.ThrowsAsync<Exception>(() => storage.WriteAsync(
+            await Assert.ThrowsAnyAsync<Exception>(() => storage.WriteAsync(
```

Comment:
Cosmos started throwing a `CosmosException` instead of `Exception` which was making this test fail.

Reply:
Can we change the test to catch CosmosException then?

Reply:
In my opinion that while it may be aggrieving, this is just test code so catching the `CosmosException` is acceptable. I agree with changing the test code to catch the `CosmosException`.